.gitignore

```diff
@@ -135,3 +135,7 @@ xontrib/__init__.py
 .copier-answers.yml
 
 pg
+
+.devenv*
+devenv*
+.envrc
```

Comment:
Can you remove these since they aren't needed for the repo? 

If you want to have git ignore files but not include them in the `.gitignore` you can add them to the `.git/info/exclude` file instead and that way they get ignored on your local computer but won't be included in the repo.